package.json

```diff
@@ -162,7 +162,7 @@
       "brotli": false,
       "path": "./packages/teams-js/dist/esm/packages/teams-js/src/index.js",
       "import": "{ app, authentication, pages }",
-      "limit": "57.33 KB"
+      "limit": "57.40 KB"
```

Comment:
Just curious, do we know why the size has increased?

Reply:
The size increase is from [this pr](https://github.com/OfficeDev/microsoft-teams-library-js/pull/2794) that merged in earlier today. It made some changes to the app file, which is one of the baseline files for the size check, so we needed a (completely valid) bump to the size check. For some reason, when the library builds in the PR, vs when it builds in the prod pipeline, there is like a 3 byte difference in the build sizes. Probably due to some string pipeline variable or something. The size limit is supposed to be a little bit higher than the current build size, but it only got increased to the PR build size. So now when it builds in the pipeline, it is 3 bytes over the size limit causing it to fail.